MekHQ/src/mekhq/resources/PersonnelTableMouseAdapter.properties

```diff
@@ -2,7 +2,8 @@ kiaQ.text=KIA?
 abilityDesc.format=%s %s
 addPregnancy.text=Add Pregnancy
 addSingleLogEntry.text=Add Single Log Entry...
-addXP.text=Add XP
+addXP.text=Add XP...
+add1XP.text=Add 1 XP
```

Comment:
This keeps popping up in all your reviews. It doesn't seem like a problem, but what I'd suggest is to use a separate branch for each pull request so that your code doesn't mix between PRs.

Reply:
Oops. I based the branches off of the fix then forgot about it.